operator/v1/types_network.go

```diff
@@ -176,7 +179,7 @@ type ProxyConfig struct {
 	BindAddress string `json:"bindAddress,omitempty"`
 
 	// Any additional arguments to pass to the kubeproxy process
-	ProxyArguments map[string][]string `json:"proxyArguments,omitempty"`
```

Comment:
is there an upstream issue or PR to fix kubebuilder?

Reply:
Not that I could find with a quick search, I'd like @ironcladlou to weigh in because I remember he might have seen something similar with kubebuilder before